docs/README.rst

```diff
@@ -95,13 +95,22 @@ Requirements
 
 * Ruby
 * Docker
+* Vagrant (FreeBSD)
 
+Linux:
 ::
 
   gem install bundler
   bundle install
   bundle exec kitchen test all
 
+FreeBSD:
+::
+
+  gem install bundler
+  bundle install
+  bundle exec KITCHEN_LOCAL_YAML=.kitchen.freebsd.yml kitchen test -t test_freebsd/integration
```

Comment:
This command doesn't work : `bundler: command not found: KITCHEN_LOCAL_YAML=.kitchen.freebsd.yml`
It should be `KITCHEN_LOCAL_YAML=.kitchen.freebsd.yml bundle exec kitchen test -t test_freebsd/integration`